src/applications/vaos/services/appointment/types.js

```diff
@@ -13,32 +13,32 @@
  * @typedef {Object} Appointment
  * @property {'Appointment'} resourceType Static resource type string
  * @property {string} id Mapped from appointment.id, request.id, or ccAppointment.id
- * @property {string} ?created Mapped from request.createdDate, timezone is unclear
- * @property {Object} ?cancelationReason Cancellation reason for a requestion, mapped from request.appointmentRequestDetailCode
+ * @property {?string} created Mapped from request.createdDate, timezone is unclear
```

Comment:
I realized I did a bunch of these incorrectly putting the nullable (`?`) character on the name instead of the type